riscv/src/continuations.rs

```diff
@@ -68,10 +70,17 @@ where
     let num_chunks = bootloader_inputs.len();
 
     log::info!("Computing fixed columns...");
-    pipeline.compute_fixed_cols().unwrap();
+    let fixed_cols = pipeline.compute_fixed_cols().unwrap();
+
+    pipeline.compute_optimized_pil().unwrap();
```

Comment:
```suggestion
    // Advance the pipeline to the optimized PIL stage, so that it doesn't need to be computed
    // in every chunk.
    pipeline.compute_optimized_pil().unwrap();
```